release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -292,7 +292,7 @@ Note the following restrictions for {product-title} on IBM Z and LinuxONE:
 ** Automatic repair of damaged machines with machine health checking
 ** Tang mode disk encryption during {product-title} deployment
 ** {product-title} Serverless
-** Helm command-line interface (CLI) tool
+**  (CLI) tool
```

Comment:
Which CLI tool? I don't think that there is enough context here.

Reply:
I am not sure what happened there may an erroneous key stroke but I don't believe I touched that section. I will replace with the correct 4.6 content - good spot.

Reply:
Thank you! (If you have a cat, I blame it.)